core/commands/files.go

```diff
@@ -321,7 +321,23 @@ func walkBlock(ctx context.Context, dagserv ipld.DAGService, nd ipld.Node) (bool
 
 var filesCpCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Copy files into mfs.",
+		Tagline: "Copy any IPFS objects into mfs (or copy within MFS).",
```

Comment:
"objects" is a bit confusing. I'd say "copy ipfs files & directories into MFS".

Reply:
I tend to agree. I have fixed this and aligned the flag descriptions (which also used "object")/